Sources/Build/BuildOperationBuildSystemDelegateHandler.swift

```diff
@@ -826,8 +826,10 @@ final class BuildOperationBuildSystemDelegateHandler: LLBuildBuildSystemDelegate
         process: ProcessHandle,
         result: CommandExtendedResult
     ) {
+        // FIXME: This should really happen at the command-level and is just a stopgap measure.
+        let shouldFilterOutput = !self.logLevel.isVerbose && command.verboseDescription.components(separatedBy: " ").first == "codesign" && result.result != .failed
```

Comment:
Could we have this use `hasPrefix` if checking the description is really the only way to check this?